server/services/userService.ts

```diff
@@ -9,12 +9,37 @@ import type { SystemToken } from '@hmpps-auth'
 import type { User } from '@manage-users-api'
 import type { Caseload } from '@prison-api'
 
+const throwUserLoadNon404Error = (username: string, knownError: ResponseError) => {
+  const errorMessage =
+    knownError.message === 'Internal Server Error' ? `Error fetching user ${username}.` : knownError.message
+  throw createError(knownError.status || 500, errorMessage)
+}
+
 export default class UserService {
   constructor(
     private readonly hmppsManageUsersClientBuilder: RestClientBuilder<HmppsManageUsersClient>,
     private readonly prisonApiClientBuilder: RestClientBuilder<PrisonApiClient>,
   ) {}
 
+  /*
+   * Rather than return a boolean this function returns a thruthy by way of the User object
```

Comment:
Looks like you've got a typo in `truthy`, unless this is a word I didn't know!

Reply:
Do we need the comment? I wonder if we could DRY up the overlap between this method and `getUserFromUsername`, if the difference is mostly just what to do in the case of a 404, depending on urgency here?

Reply:
I agree, we've not used comments much to this effect in this codebase, and it looks a bit inconsistent. Let's get rid of them here.

Reply:
> Looks like you've got a typo in `truthy`, unless this is a word I didn't know!

It's programming jargon for anything that resolves by the compiler to be true.